simapp/simd/cmd/testnet.go

```
@@ -552,46 +622,132 @@ func writeFile(name, dir string, contents []byte) error {
+
+	// channel to signal shutdown
+	shutdownCh := make(chan struct{})
+
+	fmt.Println("Starting test network...")
+	// Start each validator in a separate process
+	for i := 0; i < args.numValidators; i++ {
+		_, nodeDir := getNodeDir(args, i)
+
+		// run start command
+		cmdArgs := []string{"start", fmt.Sprintf("--%s=%s", flags.FlagHome, nodeDir)}
+		runCmd := exec.Command(os.Args[0], cmdArgs...) // spawn new process
```

golangci-lint (1.62.2)

[medium] G204: Subprocess launched with a potential tainted input or cmd arguments

(gosec)